README.md

```diff
@@ -1 +1,41 @@
-# thundra-es-plugin
\ No newline at end of file
+# thundra-es-plugin
+
+## Installation
+
+```bash
+npm install --save-dev @thundra/esbuild-plugin
+````
+
+## Usage
+#### **`esbuild.js`**
+```js
+const { ThundraEsbuildPlugin } = require('@thundra/esbuild-plugin');
+
+module.exports = {
+  ...
+  plugins: [
+    ThundraEsbuildPlugin({
+      traceableConfigs: [
+        'src.*[traceLineByLine=true]', // activate line by line tracing for all files under src folder
```

Comment:
Shouldn't it be `src.*` => `src.*.*`? Because `src.*` means *all* files under `src`, and the last `*` means all the methods